tests/test.rs

```diff
@@ -107,24 +107,28 @@ mod tests {
         lines
     }
 
-    fn setup_chainman_with_blocks(context: &Arc<Context>, data_dir: &str) -> ChainstateManager {
+    fn setup_chainman_with_blocks(
+        context: &Arc<Context>,
+        data_dir: &str,
+    ) -> Result<ChainstateManager, KernelError> {
         let blocks_dir = data_dir.to_string() + "/blocks";
         let block_data = read_block_data();
 
-        let chainman = ChainstateManager::new(
-            ChainstateManagerOptions::new(context, data_dir, &blocks_dir).unwrap(),
-        )
-        .unwrap();
+        let chainman = ChainstateManager::new(ChainstateManagerOptions::new(
```

Comment:
```rust 
let options = ChainstateManagerOptions::new(context, data_dir, &blocks_dir)?;
let chainman = ChainstateManager::new(options)?;
```
nit: nicer to avoid the `)?)?;` I think

Reply:
Can you submit it? Busy with another Core rebase atm.

Reply:
Done in https://github.com/TheCharlatan/rust-bitcoinkernel/pull/87.